README.md

```diff
@@ -241,6 +242,10 @@ whether the device is running on a simulator.
 var isSim = device.isVirtual;
 ```
 
+## device.sdkVersion (Android only)
+
+Will return the Android device's installed SDK version.
```

Comment:
The use of `installed` to me seems like this is something configurable such as the use of a target SDK, rather than the device's running SDK version. Perhaps we can scratch the word `installed` altogether?

```suggestion
Will return the Android device's SDK version.
```

Reply:
Ahh! I missed that, despite my changes! Great pickup... thanks @breautek